web/pom.xml

```diff
@@ -28,6 +28,12 @@
       <groupId>org.kontinuity.catapult</groupId>
       <artifactId>catapult-core-impl</artifactId>
     </dependency>
+    <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-multipart-provider</artifactId>
+      <version>${version.org.jboss.resteasy_resteasy.multipart.provider}</version>
```

Comment:
Just realized that this dependency is provided in the parent pom. Swarm needs the jaxrs-multipart fraction (which doesn't seem to be detected - bug?). So perhaps it would make sense to declare the needed fractions explicitly